test/losses/test_losses.py

```diff
@@ -753,3 +753,227 @@ def test_module(self, device, dtype):
         op_module = kornia.losses.LovaszSoftmaxLoss()
 
         assert_close(op(logits, labels), op_module(logits, labels))
+
+
+class TestWelschLoss:
```

Comment:
```suggestion
class TestWelschLoss(BaseTester):
```

`from kornia.testing import BaseTester`

Reply:
Done